docs/wiki/99 - Developers/UnitTestGuide.md

```diff
@@ -1,19 +1,8 @@
 # **Unit Testing 101: How to Write Tests for LORIS**
 
+This guide is for [LORIS](http://loris.ca/) developers or anyone working with LORIS who wants to contribute unit tests to the [LORIS codebase](https://github.com/aces/Loris) and learn best practices for setting up and writing unit tests. It was originally created as part of [Google Summer of Code 2019 by Alexandra Livadas](https://alexandralivadas.github.io/pages/gsoc-2019.html)) for the LORIS 21 release.  
```

Comment:
Would you mind deleting these lines? It's unusual for us to include author info in documentation like this. And if anybody is on this page they probably already know about LORIS and where to find the codebase. 

Your authorship will be forever tracked by git and if anybody challenges you on this I'm happy to back you up. :)